src/style.css

```diff
@@ -1 +1,96 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+  local("Avenir"),
+  url(fonts/Avenir.ttc),
+  url(fonts/Avenir-Book.ttf),
+  url(fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: flex;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)